cmd/lint.go

```diff
@@ -224,6 +240,14 @@ func lint(args []string, params lintCommandParams) (report.Report, error) {
 		regal = regal.WithIgnore(params.ignoreFiles.v)
 	}
 
+	if params.metrics {
+		regal = regal.WithMetrics(m)
+	}
+
+	if params.metrics {
+		m.Timer(regalmetrics.RegalConfigParse).Start()
+	}
```

Comment:
[nit]
```suggestion
	if params.metrics {
		regal = regal.WithMetrics(m)
		m.Timer(regalmetrics.RegalConfigParse).Start()
	}
```

Reply:
Ah, yes, of course! Thanks, will fix before merge.